src/Controller/CallbackAction.php

```diff
@@ -150,4 +155,22 @@ private function getUserProfileFromTokenClaims(array $tokenClaims): string
 
         return $tokenClaims['firstname'].' '.$tokenClaims['lastname'];
     }
+
+    private function createDemoCatalog(string $pimUrl, string $accessToken): string
```

Comment:
maybe add a `todo: use akeneo api php client when this endpoint will available` ?